source/features/dim-bots.tsx

```diff
@@ -83,6 +88,9 @@ void features.add(import.meta.url, {
 Test URLs
 
 - Commits: https://github.com/typed-ember/ember-cli-typescript/commits/master?after=5ff0c078a4274aeccaf83382c0d6b46323f57397+174
+- Commits by unmarked bot: https://github.com/rust-lang/rust/commits/master?after=c387f012b14a3d64e0d580b7ebe65e5325bcf822+34&branch=master&qualified_name=refs%2Fheads%2Fmaster
 - PRs: https://github.com/OctoLinker/OctoLinker/pulls?q=is%3Apr+is%3Aclosed
+- PRs by unmarked bot: https://github.com/spotify/scio/pulls?q=is%3Apr+sort%3Aupdated-desc+is%3Aclosed+steward
+- PR Commits: https://github.com/pixiebrix/webext-messenger/pull/173/commits
```

Comment:
I added more test URLs, we should almost always use these when adding test URL to a new PR